airbyte-ci/connectors/cdk/cdk/java_cdk/ci.py

```diff
@@ -0,0 +1,81 @@
+
+from typing import Awaitable, List, Optional
+
+from aircmd.models.base import PipelineContext
+from aircmd.models.click_commands import ClickCommandMetadata, ClickFlag, ClickGroup
+from aircmd.models.click_params import ParameterType
+from aircmd.models.utils import make_pass_decorator
+from dagger import Client, Container
+from prefect import flow
+from prefect.utilities.annotations import quote
+
+from .settings import JavaCDKSettings
+from .tasks import build_java_cdk_task, test_java_cdk_task
+
+pass_pipeline_context = make_pass_decorator(PipelineContext, ensure=True)
+pass_global_settings = make_pass_decorator(JavaCDKSettings, ensure=True)
```

Comment:
These are BaseSettings classes that derive information from the environment, with default values. They are singletons and intended to be immutable.